frettaApp

```diff
@@ -1 +0,0 @@
-Subproject commit 73ca35c584f0427e002830ef4716e2092353242a
```

Comment:
when I ran `git rm -r --cached .` this was also removed, so there is something in the gitignore file that says this should not be on github (I think we can trust that, otherwise I don't know what file this is or what it does, and I think it doesn't matter at all)